src/index.css.js

```diff
@@ -1,3 +1,4 @@
+export default /* css */ `
```

Comment:
The `/* css */` tag on this template helps with syntax highlighting. I'm using the following plugin in VSCode.

https://marketplace.visualstudio.com/items?itemName=pushqrdx.inline-html